src/libraries/Microsoft.PowerFx.Interpreter/Environment/RuntimeConfig.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx
+{
+    // This should have read-only views on all members. 
+    // Assume any field is nullable. 
+    public interface IRuntimeConfig
+    {
+        /// <summary>
+        /// This should match the SymbolTable provided at bind time. 
+        /// </summary>
+        public ReadOnlySymbolValues Values { get; }
+
+        /// <summary>
+        /// Services. 
+        /// </summary>
+        public IServiceProvider Services { get; }
```

Comment:
>Services [](http://example.com/codeflow?start=32&length=8)

Should be named ServiceProvider as it implements an IServiceProvider

Reply:
ok